app/styles/partials/overrides.scss

```diff
@@ -35,16 +35,38 @@ body.dimmable.undetached.dimmed {
   }
 }
 
-.margin.less {
-  padding: 0 !important;
+.less {
```

Comment:
@niranjan94 
Is this correct? 
`.less {
  &.margin {
    margin: 0 !important;
  }

  &.bottom.margin {
    margin-bottom: 0 !important;
  }

  &.top.margin {
    margin-top: 0 !important;
  }
}

- 
`